README.md

```diff
@@ -0,0 +1,38 @@
+# Pencil
```

Comment:
It was more about keeping the project less visible to search, until we test it on production properly.